packages/roosterjs-editor-api/lib/utils/toggleListType.ts

```diff
@@ -11,8 +11,19 @@ export default function toggleListType(
     listType: ListType.Ordered,
     startNumber: number
 ): void;
+export default function toggleListType(
```

Comment:
All these 3 overrides can compatible with each other, so no need to have 3 different overrides, just one override with optional parameter should work fine.

Reply:
I mean, we can delete line 8 - line 18

Reply:
And now that the function is exported, we need a regular comment, and remove "@intenal

Reply:
And the comment need to reflect the default value of includeSiblingLists is true